components/Tabs/Tabs.js

```diff
@@ -1,88 +1,94 @@
 class TabCard {
-  constructor(element){
+  constructor(element) {
     // Use a jQuery selector to assign this.element to the DOM reference
-    this.element;
+    this.element = $(element);
   }
-  selectCard(){
+  selectCard() {
     // show the card
-    this.element;
+    this.element.addClass('card');
   }
-  deselectCard(){
+  deselectCard() {
     // hide the card
-    this.element;
+    this.element.removeClass('card');
   }
 }
 
 class TabLink {
-  constructor(element, parent){
+  constructor(element,parent) {
     // Use a jQuery selector to assign this.element to the DOM reference
-    this.element;
+    this.element = $(element);
     // assign this.parent to the parent parameter
-    this.parent;
+    this.parent = $(parent);
 
     // Note that we are calling getCards() on Tabs (The parent of TabLink) and passing the data attribute: data-tab, no need to update this line of code.
-    this.cards = this.parent.getCards(this.element.data('tab'));
+    this.cards = this.parent.getCards(this.element.data('tabs'));
```

Comment:
this line should have stayed as original: `this.cards = this.parent.getCards(this.element.data('tab'));` Remember to fully read instructions.